IFC4x3/Properties/p/PostHeight_2$hlD6trH6Dh9ylUnZ4j7z/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="PostHeight_2hlD6trH6Dh9ylUnZ4j7z" Name="PostHeight" UniqueId="bfaef346-df54-4636-b27c-bdec6312d1fd" PrimaryDataType="IfcPositiveLengthMeasure" />
```

Comment:
Is this not possible to get from geometry? Otherwise, it seems a candidate for a quantity.

Reply:
Removed: https://github.com/bSI-InfraRoom/IFC-Specification/pull/179/commits/8fabd1060bd20c1e90fa52ab338868cf525b25f7